### DISCUSSION TITLE
GPU types for running monailabel server

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1401] [Q&A]
Hi!

Which GPU types support running monailabel server? Can I use RTX6000?

I don't see the GPU requirements in the installation doc [here](https://docs.monai.io/projects/label/en/latest/installation.html#prerequisites).

**Comment by tangy5** [comment]
Yes, RTX6000 is a very good GPU, currently, we don't have a specific list of available GPU as supported matrix for the open-sourced platform. Users can use monailabel across different machine with customized support. 

But from our AI enterprises support matrix, I saw monailabel supports most of Nvidia GPUs, for monailabel smooth experience, a minimum of 12G memory is recommended. For example, we have some larger models, the HoVerNet mode for pathology, whole body/whole brain, swin unetr, etc that requires more memory, those models are trained with high resolution images and V100/A100 GPUs. 

:)

**Reply by hw-ju** [reply]
Gotcha! Thanks for the info @tangy5!
